android/app/build.gradle

```diff
@@ -38,7 +39,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.example.society_app_mobile"
+        applicationId "com.programming_society.society_app_mobile"
```

Comment:
Instead of programming_society.society_app_mobile, can we change it to programming.society_app_mobile?

Reply:
No we can't it's a one time change with firebase, it's locked in unfortunately. However the id is an internal thing only and is never publicly facing.